### DISCUSSION TITLE
BUG： "&lt;&gt;"

### DISCUSSION THREAD

**Discussion by windkwbs** [open-webui/open-webui#3592] [General]
When the AI generates the "&lt;&gt;" symbol in a conversation, it is transformed into "<>" by the browser, causing the intended display of "&lt;br&gt" to become "<br>". Can this issue be resolved to ensure the browser displays the "&lt;&gt;" symbol correctly?

**Comment by Algorithm5838** [comment]
Press edit on the LLM's response and check: are they written between two sets of triple backticks (```)?

**Reply by windkwbs** [reply]
yes

**Reply by Algorithm5838** [reply]
The issue appears to be a known bug [(#2603)](https://github.com/open-webui/open-webui/issues/2603). Symbols intended to be displayed within a code block are incorrectly rendering as inline code [(#3322)](https://github.com/open-webui/open-webui/discussions/3322).
